test/mailers/business_mailer_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+class BusinessMailerTest < ActionMailer::TestCase
+  test "calling welcome enqueues the mailer" do
+    BusinessMailer.with(business:).welcome.deliver_later
+    assert_enqueued_emails 1
+  end
+
+  def business
+    businesses(:one)
+  end
+
+  def recipient
+    business.user
+  end
```

Comment:
This method isn't used, let's remove it.

Reply:
When changing the assertion to match the `developer_mailer_test`, this method ended up being used. 👍